dynamic_inventory/tasks/main.yml

```diff
@@ -27,6 +27,7 @@
     groups: ["{{ item.hosttype }}","{{ cluster_name }}","{{ clusterid }}"]
     ansible_host: "{{ item.inventory_ip }}"
     hosttype: "{{ item.hosttype }}"
+    az_name: "{{ item.az_name[-1] }}"

```

Comment:
should the `[-1]` be in `dynamic_inventory/tasks/gce.yml`?

Reply:
I guess it doesn't matter, but in that case, should be in the text inventory below?

Reply:
in this way it will cover GCP and AWS